### ISSUE TITLE
‘async’ is a reserved word in Python >= 3.7

### ISSUE THREAD

**Issue by cclauss** [mmirman/MentisOculi#2]
Use __non_blocking__ instead.

flake8 testing of https://github.com/mmirman/MentisOculi on Python 3.7.1

$ __flake8 . --count --select=E901,E999,F821,F822,F823 --show-source --statistics__
```
./helpers.py:34:85: E999 SyntaxError: invalid syntax
    dtype = lambda *args, **kargs: torch.cuda.DoubleTensor(*args, **kargs).cuda(async=cuda_async)
                                                                                    ^
1     E999 SyntaxError: invalid syntax
1
```

**Comment by mmirman** [comment]
Thanks for the heads up, I will fix this!

**Comment by mmirman** [comment]
Closed by ce0ad44
